Automattic/jetpack
# Premium Blocks: Set premium icons for jetpack premium blocks

This PR sets premium icons for Premium Blocks defined in Jetpack.
~**This PR depends on https://github.com/Automattic/jetpack/pull/16601 so review it before starting with this.**~

### 🌱 Pointing to a Feature branch

This PR is referenced to the [`feature/premium-blocks ` feature branch](https://github.com/Automattic/jetpack/pull/16611).

Fixes #

#### Changes proposed in this Pull Request:

* Set icons for Jetpack Premium Blocks.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Apply these changes in your sandbox (D47023-code)
* Test in a Simple site with a Free Plan
* Sandbox the testing site.
* Check that the following premium blocks show a little red circle indicating they are premium.

<img src="https://user-images.githubusercontent.com/77539/88602197-a86a8f80-d048-11ea-90dc-3ebbc79a91f6.png" width="400px" />



* Set icons for Jetpack Premium Blocks.
